yebeen edit

hyb26 opened the pull request:
comment, notification, place 수정

hyb26 committed:
:sparkles: [Feat] Comment 관련 Controller, Service, Repository, Mapper 추가 (작성, 전체조회, 삭제, 상세 조회)

hyb26 committed:
:sparkles: [Feat] Notification 관련 Controller, Service, Repository, Mapper 추가 (전체조회, 읽음처리, 알림 갯수 반환)

hyb26 committed:
:sparkles: [Feat] Place 관련 entity(place, heart, placeimg), dto 수정 및 추가

hyb26 committed:
:sparkles: [Feat] Place 관련 entity(place, heart, placeimg), dto, controller 수정, Comment 및 Notification 수정

seungchul-lee118 commented on sogo/src/main/java/com/enzinior/sogo/comment/dto/CommentDto.java:
```diff
@@ -0,0 +1,35 @@
+        @NotBlank
+        private String userUuid;
+        @NotBlank
+        private String parentUuid;
```
Parent가 nullable로 설정해서 null / null이 아닌 값 으로 구분하기로 했던 것으로 기억하는데
@NotBlank 로 Validation을 하면 null이 못들어와서 @NotBlank 빼야할 것 같음

seungchul-lee118 commented on sogo/src/main/java/com/enzinior/sogo/comment/mapper/CommentMapper.java:
```diff
@@ -0,0 +1,24 @@
+
+    @Mapping(source = "reviewUUID", target = "review.uuid")
+    @Mapping(source = "userUUID", target = "user.uuid")
+    @Mapping(source = "parentUUID", target = "user.uuid")
```
@Mapping에 source는 CommnetDto.Post에 있는 필드명 그대로 적기 (reviewUUID -> reviewUuid)

@Mapping(source = "parentUUID", target = "user.uuid") parentUuid는 user의 parent로 매핑

hyb26 replied:
user의 parent로 매핑이라는 말이 잘 이해가 안가는데, 그 댓글이 달린 언 댓글의 uuid를 매핑할 것이니 없애라는 말씀이시죠?

seungchul-lee118 commented on sogo/src/main/java/com/enzinior/sogo/comment/mapper/CommentMapper.java:
```diff
@@ -0,0 +1,24 @@
+
+    @Mapping(source = "userNickname", target = "review.nickname")
+    @Mapping(source = "userImg", target = "user.img")
+    @Mapping(source = "parentUUID", target = "user.uuid")
```
여기도 위에 parentUuid 코멘트와 동일

hyb26 committed:
:sparkles: [Feat] 수정사항
- 피드백 주신 부분들 수정
- comment, heart, notification, place, placeimg의 entity id명 수정
- comment, notification controller에서 ResponseEntity로 return하지 않은 부분들 중 일부 수정 및 나머지 로직은 반환값 고민중(금요일 전까지 수정 예정)
- CommentServiceImpl, NotificationServiceImpl에서 @Transactional(readOnly) 추가
